draft-ietf-taps-transport-security.md

```diff
@@ -249,11 +251,63 @@ interface or address changes.
 
 - Server: the peer responsible for responding to a session initiation.
 
+# Security Features
+
+In this section, we enumerate Security Features exposed by protocols discussed in the
+remainder of this document. Protocol security properties that are unrelated to the API surface
+exposed by such protocols, such as client or server identity hiding, are not listed 
+here as features.
+
+- Forward-secure key establishment: Cryptographic key establishment with forward secure properties.
+
+- Cryptographic algorithm negotiation: Negotiate support of protocol algorithms, including: encryption, 
+hash, MAC (PRF), and digital signature algorithms.
+
+- Stateful and stateless cross-connection session resumption: Connection establishment without needing 
+to complete an entirely new handshake. 
+
+- Peer authentication (certificate, raw public key, pre-shared key, or EAP-based): Peer authentication using 
+select or protocol-specific mechanisms.
```

Comment:
select?

Reply:
I couldn't come up with a better definition this morning. So I'm all ears if you've a suggestion!

Reply:
Why not just protocol-specific? What is the or case covering?